x/observer/types/events.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	BallotIdentifier       = "BallotIdentifier"
+	CCTXIndex              = "CCTXIndex"
+	BallotObservationHash  = "BallotObservationHash"
+	BallotObservationChain = "BallotObservationChain"
+	BallotType             = "BallotType"
+	BallotCreated          = "BallotCreated"
+)
+
+func EmitEventBallotCreated(ctx sdk.Context, ballot Ballot, observationHash, obserVationChain string) {
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(BallotCreated,
+			sdk.NewAttribute(BallotIdentifier, ballot.BallotIdentifier),
+			sdk.NewAttribute(CCTXIndex, ballot.BallotIdentifier),
+			sdk.NewAttribute(BallotObservationHash, observationHash),
+			sdk.NewAttribute(BallotObservationChain, obserVationChain),
+			sdk.NewAttribute(BallotType, ballot.ObservationType.String()),
+		),
+	)
+}
```

Comment:
We already have a typed event for Ballot created .

Reply:
https://github.com/zeta-chain/zeta-node/blob/452206ce6fa67e4364cac519bdbcd5d2e7833589/x/observer/keeper/events.go#L11-L21

Reply:
Yes, this is the same one that I moved to observer

Reply:
I see, it was moved to the keeper instead. I'll remove it